fio/fio.py

```diff
@@ -39,9 +39,14 @@ def run_once(self, opts=None, job=None, user='root'):
 
         if opts:
             _opts += opts
+        arch_output = commands.getoutput("arch")
```

Comment:
we can use platform module to get host arch.
code like:
import platform
arch = platform.machine()
job_file = "fio-mixed.job"
if "ppc64" in arch:
     job_file = 'fio-mixed-ppc.job'